docs/guides/getting-started-java-spring.md

```diff
@@ -204,14 +209,14 @@ public class ProcessPaymentsApplication implements CommandLineRunner {
 ```java
 	@Override
 	public void run(final String... args) {
-		var processDefinitionKey = "process-payments";
+		var bpmnProcessId = "process-payments";
```

Comment:
@pepopowitz What are your thoughts on this? Does this also mean I need to change line 214?

Reply:
Based on your feelings here I'll adjust the actual reference in the GH repo.

Reply:
yes, definitely!